src/Altinn.AccessManagement/Controllers/MaskinportenSchemaController.cs

```diff
@@ -131,9 +129,8 @@ public async Task<ActionResult<DelegationOutputExternal>> MaskinportenScopeDeleg
                 }
 
                 DelegationOutputExternal delegationOutput = _mapper.Map<DelegationOutputExternal>(response);
-                DelegationHelper.TryGetResourceFromAttributeMatch(response.Rights.First().Resource, out var _, out string resourceId, out var _, out var _);
-                DelegationHelper.TryGetPartyIdFromAttributeMatch(internalDelegation.To, out int toPartyId);
-                return Created(new Uri($"https://{Request.Host}/accessmanagement/api/v1/{party}/delegations/maskinportenschema/offered?to={toPartyId}&resourceId={resourceId}"), delegationOutput);
+                
+                return Created(new Uri($"https://{Request.Host}/accessmanagement/api/v1/{party}/delegations/maskinportenschema/offered?to={delegationOutput.To.First().Value}&resourceId={delegationOutput.RightDelegationResults.First().Resource.First().Value}"), delegationOutput);
```

Comment:
Just remove the Created URI completely, as it's wrong anyway.
There is no Get endpoint for retrieving the delegation with URL params

Reply:
Instead of using the Created method you can just use the StatusCode metod

Reply:
Fixed